test/either-spec.js

```diff
@@ -311,6 +311,24 @@ describe('An Either', function () {
         })
     })
 
+    describe('Either.isInstance', function () {
+        it('will return true only for Either instances', function () {
```

Comment:
nitpicking but I believe the test description should not contain the word "only" here, this particular test does not test negative cases